packages/sync/src/modules/Full_Sync.php

```diff
@@ -217,6 +217,8 @@ public function continue_enqueuing( $configs = null, $enqueue_status = null ) {
 				$modules_processed ++;
 				continue;
 			}
+			// Recalculate the total for some modules.
+			$enqueue_status[ $module_name ][0] = $module->recalculate_total( $enqueue_status[ $module_name ] );
```

Comment:
isn't this the total at the beginning of the full sync?

Reply:
Yes this is the total. We are just recalculating it for every `continue_enqueuing`